### DISCUSSION TITLE
How does maplibre-rs compare to maplibre-gl-native

### DISCUSSION THREAD

**Discussion by ueoeae** [maplibre/maplibre-rs#98] [Q&A] [answered]
Is maplibre-rs an alternative to maplibre-gl-native? And is windows support planned?

**Comment by maxammann** [comment, marked as answer]
That question is still a little bit undecided. `maplibre-rs` is a cross-platform vector tile renderer. Whether a goal is to reach feature-parity and complete style.json support is questionable.

We can also use maplibre-rs to test new methods.

Which kind of windows support do you imaging? Would you like to be able to embed maplibre-rs in Qt applications? Woud you like to run it standalone?
@Drabble is working on testing this on windows. I think compilation already worked in WSL.

**Reply by ueoeae** [reply]
Thanks for the swift reply.

Currently using `maplibre-gl-native` to render vector and raster tiles. Since, if I remember correctly only QT compilation is _somewhat_ supported, I managed to compile it on windows standalone without QT, hence a standalone approach is what I am looking for.

**Reply by maxammann** [reply]
Do you mean you managed to compile maplibre-rs  standalone on windows or maplibe-gl-native?

It will proabbly take 1-2 years until maplibre-rs reaches a competitive state :)
